code/game/gamemodes/miniantags/abduction/abduction.dm

```diff
@@ -184,6 +184,7 @@
 			var/obj/machinery/abductor/console/con = get_team_console(team_number)
 			var/datum/objective/objective = team_objectives[team_number]
 			if(con.experiment.points >= objective.target_amount)
+				priority_announcement.Announce("Regular crew brainwave monitoring has detected large amount of abnormal thought patterns, heavy abductor presence suspected. All crew are recalled for mandatory physical and mental evaluation and reconditioning.")
```

Comment:
`Large amount of abnormal thought patterns detected. All crew are recalled for mandatory evaluation and reconditioning. `

seems better imo